pkg/broker/config/config.go

```diff
@@ -86,6 +86,12 @@ func BrokerKey(namespace, name string) string {
 	return namespace + "/" + name
 }
 
+// BrokerPath returns the path to be set in the status of a broker.
+// The format is brokerNamespace/brokerName
+func BrokerPath(namespace, name string) string {
+	return fmt.Sprintf("/%s/%s", namespace, name)
+}
+
```

Comment:
The name of the file might appear confusing but the actual purpose of this file for broker data plane configuration.

My suggestion is to create a `pkg/broker/ingress/path.go` file. Put this func there. Also, move this logic (https://github.com/google/knative-gcp/blob/master/pkg/broker/ingress/handler.go#L123-L133) there. And change other places accordingly.